src/sentry/conf/server.py

```diff
@@ -3781,12 +3781,13 @@ def build_cdc_postgres_init_db_volume(settings: Any) -> dict[str, dict[str, str]
 # tests)
 SENTRY_METRICS_INDEXER_RAISE_VALIDATION_ERRORS = False
 
-# The project ID for SDK Crash Detection to save the detected SDK crashed to.
-# Currently, this is a single value, as the SDK Crash Detection feature only detects crashes for the Cocoa SDK.
-# Once we start detecting crashes for other SDKs, this will be a mapping of SDK name to project ID or something similar.
-SDK_CRASH_DETECTION_PROJECT_ID: Optional[int] = None
+from sentry.conf.types.sdk_crash_detection_config import SDKCrashDetectionConfig
+
+# The configuration for SDK Crash Detection. Add one entry per SDK.
+SDK_CRASH_DETECTION: list[SDKCrashDetectionConfig] = []
```

Comment:
Is there any reason to use sentry configs as feature flags?

Reply:
@iker-barriocanal, only type safety. I can also make it a dict, if you prefer.

Reply:
Sorry for the confusion, I meant feature flags that can be controlled without modifying the sentry config. For example, using https://github.com/getsentry/sentry-options-automator, you don't have to modify the actual sentry config and can change values in production without another deployment.

Reply:
I started using the sentry config when I built this feature. I can change it to the options automaton, as that seems to be the preferred way to do this now. Not a problem.

Reply:
It's ok with me, I was just curious 😄

Reply:
if you're going to change this often the options automator is a better solution -- putting it in `conf` gives ~no benefits as well unless you mean to make it different per environment

Reply:
I moved the settings to the options.